js/masters/indicators/momentum.src.js

```diff
@@ -11,4 +11,4 @@
  * License: www.highcharts.com/license
  */
 'use strict';
-import '../../Stock/Indicators/MomentumIndicator.js';
+import '../../Stock/Indicators//Momentum/MomentumIndicator.js';
```

Comment:
Not sure if it is a problem on any platform, but to make sure the double slash should be fixed.

Reply:
True! I missed that. Thanks for spotting that